aws-athena/ctas/proximity-crosswalk_year_fill.sql

```diff
@@ -20,6 +20,8 @@ WITH unfilled AS (
             AS num_school_data_year,
         MAX(cnt_pin_num_school.num_school_rating_data_year)
             AS num_school_rating_data_year,
+        MAX(dist_pin_to_airport.airport_data_year) --new
+            AS airport_data_year,
```

Comment:
Let's drop the comments from the final PR:

```suggestion
        MAX(dist_pin_to_airport.airport_data_year)
            AS airport_data_year,
```